src/org/exist/protocolhandler/embedded/EmbeddedUpload.java

```diff
@@ -115,17 +116,11 @@ public void stream(XmldbURL xmldbURL, File tmp) throws IOException {
      * @param tmp Document that is inserted.
      * @throws IOException
      */
-    public void stream(XmldbURL xmldbURL, File tmp, Subject user) throws IOException {
+    public void stream(XmldbURL xmldbURL, Path tmp, Subject user) throws IOException {
```

Comment:
Ah right, sorry I cherry-picked a commit which should not have been included. I have now removed it.